main/modes/games/cGrove/Garden/cg_Grove.h

```diff
@@ -16,11 +16,17 @@
 
 #include "cg_Typedef.h"
 
+//==============================================================================
+// Externs
+//==============================================================================
+
+extern const char* cgNVSKeys[];
```

Comment:
JSYK, declare `cgNVSKeys[]` as `extern` in a `.h`, then declare it once in a `.c` file. It'll be accessible from anywhere that `.h` file is included.

Reply:
@johnnywycliffe ^

Reply:
Perfect